src/App.js

```diff
@@ -2,12 +2,60 @@ import React from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './GoodsList';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  loadAllGoods = () => {
+    getAll()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
+
+  load5Goods = () => {
+    get5First()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
+
+  loadRedGoods = () => {
+    getRedGoods()
+      .then((goods) => {
+        this.setState({ goods });
+      });
```

Comment:
You could make one function instead of these 3, that accepts callback as a parameter